.eslintrc.json

```diff
@@ -42,11 +42,11 @@
         "props": false
       }
     ],
-    "complexity": [ 2, 3 ],
-    "max-depth": [2, 3],
-    "max-len": [1, 160, 4],
+    "complexity": [ 2, 10 ],
```

Comment:
We should keep the lint rules consistent between repos as much as possible. I agree some of these rules could be updated, but it should be agreed for all repos.